docs/get-started/labs/lab-34.md

```diff
@@ -0,0 +1,70 @@
+---
```

Comment:
This is very good lab. Next time consider presenting a feature from a regular team perspective not team-admin.

Reply:
Ah yeah good one, I think I had some issues with enabling sealed-secrets for a team. Just posted it in the slack channel.

Reply:
Updated it with team-demo